chat.js

```diff
@@ -211,8 +211,9 @@ class CommandContext {
 						this.room.add((this.room.type === 'chat' ? (this.room.type === 'chat' ? '|c:|' + (~~(Date.now() / 1000)) + '|' : '|c|') : '|c|') + this.user.getIdentity(this.room.id) + '|' + message);
 						this.room.messageCount++;
 					}
+					this.room.add(`|c|${this.user.getIdentity(this.room.id)}|${message}`).update();
```

Comment:
It might, hence why Exiled emotes don't work in PMs I guess?  Shadow ban works though